app/generate_data.py

```diff
@@ -891,9 +923,9 @@ def determine_year(day):
 
 
 if __name__ == "__main__":
-    logpath = os.path.abspath(os.path.join(__file__, '..', 'logging'))
+    logpath = os.path.join(os.getcwd(), 'app', 'logging')

```

Comment:
These changes should have been a second commit, also not essential, I wasn't clear that the new code needing fixing, this stuff could wait and be done in dev.

Reply:
Not ideal but not really severe enough to need correcting.